File: Indixpert-Sept-batch2025/C Languege/function/with return-argument/task5.c
#include <stdio.h>

void prime(int);

void main()
{
    prime(3);
}

void prime(int num)
{
    int c = 0;

    for (int i = 1; i <= num; i++)
    {

        if (num % i == 0)
        {
            c++;
        }
    }
    if (c == 2)
    {
        printf("this is prime number ");
    }
    else
    {
        printf("Not prime number ");
    }
}

File: Indixpert-Sept-batch2025/Python/List/task5.py
list = []

num = int(input("Enter number of elements = "))

for i in range(num):
    n = int(input("Enter element ="))
    list.append(n)

search = int(input("Enter search number ="))

for i in range(num):
    if list[i] == search:
        print("Element found at index =",i)
        break
    else:
        print("Element not found")